Content.Server/Nutrition/EntitySystems/CreamPieSystem.cs

```diff
@@ -49,7 +49,7 @@ protected override void SplattedCreamPie(EntityUid uid, CreamPieComponent creamP
                 {
                     _puddle.TrySpillAt(uid, solution, out _, false);
                 }
-                if (foodComp.Trash.Count == 0)
+                if (foodComp.Trash.Count != 0)
```

Comment:
Just remove the if condition completely.
`foreach` will do nothing if the list is empty.

Reply:
Thank you :+1:  I wasn't so sure if removing the loop can affect other modules, since I'm new to the project